include/NovelRT/Utilities/Misc.h

```diff
@@ -11,13 +11,13 @@ namespace NovelRT::Utilities {
   class Misc {
   public:
     static const int SDL_SUCCESS = 0;
-    static inline const std::string CONSOLE_LOG_GENERIC = "NovelRT";
-    static inline const std::string CONSOLE_LOG_APP = "Application";
-    static inline const std::string CONSOLE_LOG_DOTNET = ".NET";
-    static inline const std::string CONSOLE_LOG_GFX = "GFX";
-    static inline const std::string CONSOLE_LOG_AUDIO = "Audio";
-    static inline const std::string CONSOLE_LOG_INPUT = "Input";
-    static inline const std::string CONSOLE_LOG_WINDOWING = "WindowManager";
+    static inline const char* CONSOLE_LOG_GENERIC = "NovelRT";
+    static inline const char* CONSOLE_LOG_APP = "Application";
+    static inline const char* CONSOLE_LOG_DOTNET = ".NET";
+    static inline const char* CONSOLE_LOG_GFX = "GFX";
+    static inline const char* CONSOLE_LOG_AUDIO = "Audio";
+    static inline const char* CONSOLE_LOG_INPUT = "Input";
+    static inline const char* CONSOLE_LOG_WINDOWING = "WindowManager";
```

Comment:
Why the change in type here?

Reply:
statics for non primitive types require global constructors and destructors, which Clang warns about (it increases startup and shutdown cost, among other things).
defining it as the primitive type instead makes it a true constant with no startup overhead and users can still get a `std::string` if required